src/db/mod.rs

```diff
@@ -36,7 +36,7 @@ lazy_static! {
     };
 }
 
-type DbFuture<T> = Box<Future<Item = T, Error = DbError>>;
+type DbFuture<T> = Box<Future<Item = T, Error = ApiError>>;
```

Comment:
Can we keep this as Error = DbError for now? I think I'll probably need to keep it this way when I begin plugging in the MysqlDb into Future calls (and maybe ApiError does make more sense here, I'll have to play around with it)